cli/reporter/print_report.go

```diff
@@ -76,17 +75,7 @@ func (r *reportRenderer) print() error {
 		}
 	}
 
-	if r.usePager && pager.Supported(r.pager) {
```

Comment:
can we also remove those reportRenderer props: `usePager` and `r.pager`? looks like we will not be needing those

Reply:
Yes, good call

Reply:
Code updated and ready for re-review